### ISSUE TITLE
FileUploader component accessibility issues

### ISSUE THREAD

**Issue by reeteshsingh93** [carbon-design-system/carbon#5726]
## Environment
Operating System: Mac OS Catalina 10.15.3 (19D76)
Browser, Version: Google chrome: Version 80.0.3987.87

> Automated testing tool and ruleset
DAP

## Detailed description

> What version of the Carbon Design System are you using?

Cloud Patterns and Assets Library Version: "@console/pal": "1.90.8"

> What did you expect to happen?

A default aria-label or aria-labelledby value or any option to define these values.

> What happened instead?

Dap is reporting issue with latest rule set

> What WCAG 2.1 checkpoint does the issue violate?
IBM Accessibility January 2020

## Steps to reproduce the issue

Create a file uploader component
Run DAP browser extension with IBM Accessibility January 2020 ruleset

Links to code or a live example of the bug:

https://dev.console.test.cloud.ibm.com/security-advisor#/integrations
Step 1: Click on direct connection tile a modal will popup with file uploader component.
Step 2: Execute DAP

**Comment by emyarod** [comment]
since my ibm cloud dev env dashboard does not contain the tile referenced in your steps to reproduce the issue, can you create a [reduced test case in Code Sandbox](https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/react/examples/codesandbox) for easier testing and debugging?

**Comment by dakahn** [comment]
Closing this up. We can reopen if there's still a problem and we get something we can reference for a verify
